### PR TITLE
Added UI for home page

### PR DESCRIPTION
**Pull request by adrean-hipe**
OVERVIEW

- added UI for the home page
- added logos of different sizes
- added welcome message upon login
- added header as a partial to the home
- added footer as a partial to the home
- added logout button to the header
- added basic placeholder image for post contents

NOTES

- the post placeholder will be further improved to only be shown when there are no posts contents
- bare UI for the home page, more features to be added
image

### CODE REVIEW

File: laravel-app/public/empty.png
Comment (hipe01mark) [inline laravel-app/public/empty.png:1]: Why are all of your assets not saved in the storage folder?

Ref:
https://laravel.com/docs/10.x/filesystem

Note:
Do the same thing with the other assets.

**Review by bpocrafael — APPROVED** [review]
GEFT

File: laravel-app/resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -14,23 +14,27 @@
     <link href="https://fonts.bunny.net/css?family=Nunito" rel="stylesheet">
```
Comment (hipe01mark) [inline laravel-app/resources/views/layouts/app.blade.php:14]: why <head></head>are not separated in the other file? create a new file and use @include instead.

Comment (adrean-hipe) [reply laravel-app/resources/views/layouts/app.blade.php]: Separated into files sir, used @include instead. Thank you ❤️

File: laravel-app/resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -14,23 +14,27 @@
...
 </head>
 <body>
     <div id="app">
-        <main class="py-4">
+        <main>
             @yield('content')
         </main>
     </div>
+    
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```
[the remark this answers is not in the document]
Comment (adrean-hipe) [reply laravel-app/resources/views/layouts/app.blade.php]: Separated into files sir, used @include instead. Thank you ❤️

File: laravel-app/public/empty.png
Comment (adrean-hipe) [reply laravel-app/public/empty.png:1]: Moved assets to storage folder and created link with artisan. Thank you ❤️

File: laravel-app/resources/views/layouts/scripts.blade.php [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.min.js" integrity="sha384-BBtl+eGJRgqQAUMxJ7pMwbEyER4l1g+O15P+16Ep7Q9Q+zqX6gSbd85u4mG4QzX+" crossorigin="anonymous"></script>
```
Comment (bpocrafael) [inline laravel-app/resources/views/layouts/scripts.blade.php:2]: add newline

Comment (adrean-hipe) [reply laravel-app/resources/views/layouts/scripts.blade.php]: Added newline sir. Thank you ❤️

File: laravel-app/resources/views/layouts/head.blade.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- CSRF Token -->
+    <meta name="csrf-token" content="{{ csrf_token() }}">
+
```
Comment (hipe01mark) [inline laravel-app/resources/views/layouts/head.blade.php:6]: Fix indention before merging it but I will mark this as resolved.

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇 If you are done with the last comment just mark it as resolved.

**Pull request merged**